components/QueryForm.vue

```diff
@@ -137,6 +155,11 @@ export default {
         case 'Imaging modality':
           this.modality = this.categoricalOptions[name][input];
           break;
+        case 'Neurobagel graph database':
+          if (!input.includes('All')) {
+            this.selectedNodes = input.map((el) => this.nodes[el]);
```

Comment:
```suggestion
            this.selectedNodeURLs = input.map((node_name) => this.nodes[node_name]);
```
for clarity, and if I understand right.